bot/utils/services.py

```diff
@@ -6,7 +6,7 @@
 import bot
 from bot.constants import URLs
 
-log = logging.getLogger(__name__)
+log = logging.getLogger(__name__)  # use logging.getLogger (not bot.utils.logging.get_logger) to prevent circular import
```

Comment:
Similar to other comment, although I feel we should be able to use the logger here, and using `get_logger` for it would be nice, although i'm not sure what the best way forwards with that is. This problem only seems to happen because this file is imported in the `utils/__init__.py`

Reply:
This has been resolved with the little re-write :+1: